readme.txt

```diff
@@ -3,7 +3,7 @@ Contributors: pratikchaskar
 Donate link: https://www.paypal.me/BrainstormForce
 Requires at least: 4.4
 Tags: beaver builder, page builder plugin, timed content, php based timed content
-Stable tag: 1.0.3
+Stable tag: 1.0.4
 Tested up to: 6.5
```

Comment:
@vijetaR WP 6.6 tag is not updated in this PR
Can you get this done??

Reply:
@sushmak02 I have raised separate PR for that, we can merge it 
https://github.com/brainstormforce/timed-content-for-beaver-builder/pull/28